kotlin-hslog/src/commonMain/kotlin/net/hearthsim/hslog/parser/power/SecretLogic.kt

```diff
@@ -185,6 +189,7 @@ internal class SecretLogic(val cardJson: CardJson, val console: Console) {
                     if (opponentHasRoomOnBoard(game)) {
                         exclude(game, CardId.BEAR_TRAP)
                         exclude(game, CardId.WANDERING_MONSTER)
+                        exclude(game, CardId.SHADOW_CLONE)
```

Comment:
Shadow Clone should only be excluded if the hero is attacked by a minion (unlike Wandering Monster where a hero attacking the hero would trigger). You can do this by wrapping the exclude in something like: `if(entity.tags[Entity.KEY_CARDTYPE] == Type.MINION`